docs/user_guide/installation_and_version_guide/v0.11.0.rst

```diff
@@ -0,0 +1,7 @@
+v0.11.0
+=======
+
+.. note::
+
+   v0.11.0 is not yet released. This page reflects changes on the current
```

Comment:
Is this same string removed in the v0.10.0 file?

Reply:
I copied it over from the v0.10.0 file so that should be the case

Reply:
Done!